tests/unit/models/keras/test_sampler.py

```diff
@@ -223,6 +218,34 @@ def test_ensemble_trajectory_sampler_samples_are_distinct_within_batch(diversify
     )  # distinct for two samples within same draw
 
 
+@pytest.mark.skip
+@random_seed
+def test_ensemble_trajectory_sampler_eps_broadcasted_correctly() -> None:
+    """
+    We check if eps are broadcasted correctly in diversify mode.
+    """
+    example_data = empty_dataset([1], [1])
+    test_data = tf.linspace([-10.0], [10.0], 100)
+    test_data = tf.expand_dims(test_data, -2)  # [N, 1, d]
+    test_data = tf.tile(test_data, [1, 2, 1])  # [N, 2, D]
+
+    model, _, _ = trieste_deep_ensemble_model(example_data, _ENSEMBLE_SIZE)
+
+    trajectory_sampler = DeepEnsembleTrajectorySampler(model, diversify=True)
+    trajectory = trajectory_sampler.get_trajectory()
+
+    _ = trajectory(test_data)  # first call needed to initialize the state
+    trajectory._eps = tf.constant([[0], [1]], dtype=tf.float64)  # type: ignore
```

Comment:
```suggestion
    trajectory._eps.assign(tf.constant([[0], [1]], dtype=tf.float64))
```

Reply:
thanks a lot, quite obvious now that you pointed it out :)